notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Copyright 2023 Google LLC
</pre><p>Change to 2025 please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>